MyRobotArmMoveitJoints joints_input
---
moveit_msgs/RobotTrajectory trajectory